frontend/src/components/ItemPage.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import {ItemModel} from "../model/ItemModel";
+import ItemNavigation from "./ItemNavigation";
+import ItemTable from "./ItemTable";
+import "../css/TablePage.css"
+import Alert from "@mui/material/Alert";
+
+function ItemPage() {
+
+    const [data, setData] = useState<ItemModel[]>([]);
+    const [errorModal, setErrorModal] = useState(false)
+    const [errorMessage, setErrorMessage] = useState("")
```

Comment:
Statt eines boolean und einer message könntest Du auch eine einzigen State nutzen. Dieser wäre dann entweder undefined (Modal geschlossen) oder ein Text (Modal geöffnet und zeigt den Text).
```suggestion
    const [errorMessage, setErrorMessage] = useState<string>()
```